packages/core/src/components/overlay/overlay.tsx

```diff
@@ -109,6 +109,15 @@ export interface IOverlayableProps extends IOverlayLifecycleProps {
      */
     portalContainer?: HTMLElement;
 
+    /**
+     * A list of DOM events which should be stopped from propagating through the Portal.
+     * This prop is ignored if `usePortal` is `false`.
+     *
+     * @see https://legacy.reactjs.org/docs/portals.html#event-bubbling-through-portals
+     * @see https://github.com/palantir/blueprint/issues/6124
+     */
+    portalStopPropagationEvents?: Array<keyof HTMLElementEventMap>;
```

Comment:
adjusted the name of this prop based on the other "portal" props on this interface. I admit it's getting a bit unwieldy and a `portalProps` prop might be better, but this is OK for now...